From 5e6638d9c32a223c2e9be4190de78f5afee65cdc Mon Sep 17 00:00:00 2001
From: Dan Prince <dprince@redhat.com>
Date: Fri, 17 Aug 2012 11:50:33 -0400
Subject: [PATCH] Update pip-requires with warlock<2.

Allow warlock to be used up to version 2 (the next
major version of the library).

Change-Id: I0c5a47f9ebfa0145dfab7310a22982d5d8e9aa52
---
 tools/pip-requires |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/pip-requires b/tools/pip-requires
index a1f3536..bbb4edf 100644
--- a/tools/pip-requires
+++ b/tools/pip-requires
@@ -1,4 +1,4 @@
 argparse
 prettytable>=0.6,<0.7
 python-keystoneclient>=0.1.2,<0.2
-warlock==0.1.0
+warlock<2
